editor/src/components/editor/store/editor-state.ts

```diff
@@ -256,8 +229,7 @@ export interface UserState extends UserConfiguration {
 export type GithubOperation =
   | { name: 'commish' }
   | { name: 'listBranches' }
-  | { name: 'loadBranch'; branchName: string; githubRepo: GithubRepo }
-  | { name: 'loadRepositories' }
+  | { name: 'loadBranch'; branchName: string }
```

Comment:
It looks like some of the newer GH code was lost during a merge. I think the only code in this file coming from your branch is around the `export enum LeftMenuTab` above, so everything below that point should be unchanged